backend/api/tests/test_admin.py

```
@@ -201,4 +205,175 @@ def test_reset_access_count_action(self):
+        # Set up the admin action request
+        request = self.create_request_with_messages()
+        
+        # Execute the action
+        queryset = Video.objects.filter(id=self.video.id)
+        self.video_admin.check_privacy_limits(request, queryset)
+        
+        # Verify mocks were called correctly
+        mock_should_make_private.assert_called_once_with(self.video)
+        mock_make_private.assert_called_once_with(self.video)
+    
+    def test_export_video_data(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_export_video_data`

Add return type annotation: `None`

(ANN201)